RocketChat/Rocket.Chat.ReactNative
# Chore: upgrade react navigation to 6.0.10 version

## Proposed changes
Upgrade react navigation.

https://reactnavigation.org/docs/upgrading-from-5.x/#elements-library

## Issue(s)	


## How to test or reproduce
As we update the library responsible to navigate between screens, we should test:
* The navigation between screens;
* Modal screens, eg: "New Message", "Edit Status", "Enter e2e password"; 
* Rooms with long names; 
* Go back buttons;

It's also necessary to test on tables and Ipads. 

## Screenshots

## Types of changes




## Checklist



## Further comments
